.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# `genomic-medicine-sweden/gms16s`: Contributing Guidelines
+
+Hi there!
+Many thanks for taking an interest in improving genomic-medicine-sweden/gms16s.
+
+We try to manage the required tasks for genomic-medicine-sweden/gms16s using GitHub issues, you probably came to this page when creating one.
+Please use the pre-filled template to save time.
+
+However, don't be put off by this template - other more general issues and suggestions are welcome!
+Contributions to the code are even more welcome ;)
+
+## Contribution workflow
+
+If you'd like to write some code for genomic-medicine-sweden/gms16s, the standard workflow is as follows:
+
+1. Check that there isn't already an issue about your idea in the [genomic-medicine-sweden/gms16s issues](https://github.com/genomic-medicine-sweden/gms16s/issues) to avoid duplicating work. If there isn't one already, please create one so that others know you're working on this
```

Comment:
Update the name and corresponding link here.

Reply:
Fixed!